install/windows/build.py

```diff
@@ -40,13 +40,13 @@ def add_size_to_version():
 
 
 def build_odemisviewer_exe():
-    rc = run_command(cpy_command)
-    return rc or run_command(pyi_command, "odemis")
+    run_command(cpy_command)
```

Comment:
That changes the behaviour. The original behaviour is that if the first command returns non-zero, it will immediately return that value, otherwise, it'll run the second command and return the value.
That said, it's a really weird way to write it. This is hopefully easier to read:
```
rc = run_command(cpy_command)
if rc:
     return rc
return run_command(pyi_command, "odemis")
```

Reply:
Does run_command return anything? There is no return...

Reply:
Oh.... I see now... I had assumed that `run_command` does the "normal" behaviour of executing a program, and returns 0 if there is no error.
Instead, it already handle the error by itself, and quit the python program by itself. So you could just write:
```
run_command(...)
run_command(...)
```
(not even a `return`)